examples/docker/README.md

```diff
@@ -2,16 +2,58 @@
 
 ![](https://github.com/artipie/artipie/workflows/Proof::docker/badge.svg)
 
-Try this `docker.yaml` file:
+This directory contains a basic example of how can Artipie be used as a Docker registry.
+Try this example by running `run.sh` script.
+
+Basic configuration `my-docker.yaml`:
 
 ```yaml
 repo:
   type: docker
   storage:
     type: fs
-    path: /tmp/artipie/data/my-docker
+    path: /var/artipie/data
+```
+
+After creating the configuration file below, Artipie is ready to server as a Docker registry.
+
+Before pushing any images let's pull an existing one from Docker Hub:
+
+```bash
+docker pull ubuntu
 ```
 
+Since the Docker registry is going to be located at localhost:8080/my-docker, we tag the pulled
```

Comment:
@Sammers21 I'd suggest to use `as` here instead of `since` (fill free to ignore if you disagree)